tests/src/JIT/HardwareIntrinsics/X86/Regression/GitHub_21666/GitHub_21666.cs

```diff
@@ -0,0 +1,220 @@
+using System;
+using System.Runtime.Intrinsics.X86;
+using System.Runtime.Intrinsics;
+
+namespace GitHub_21666
+{
+    class Program
+    {
+        const int Pass = 100;
+        const int Fail = 0;
+
+        static byte byteSF = 1;
+        static ushort ushortSF = 1;
+        static uint uintSF = 1;
+        static ulong ulongSF = 1;
+
+        readonly static byte[] byteArray = new byte[10];
+        readonly static ushort[] ushortArray = new ushort[10];
+        readonly static uint[] uintArray = new uint[10];
+        readonly static ulong[] ulongArray = new ulong[10];
```

Comment:
Nit: as per the [coding guidelines](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) pt.3, `private static` fields will have `s_` prefix (line 12-20).